### PR TITLE
Jordan/2170 explorer mode fixes

### LINKED ISSUE #2170: Improve explorer mode
**Describe the bug**

the wallet page is still blank and the tx page shows no transactions instead of the sign in card if the user not signed in
Do we even need to prompt right away for sign in?

**To Reproduce**

Sign in
Go to Wallet

**Expected behavior**

Show sign in card

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by jbibla**
Closes #2170 

**Description:**

- fixed bug in card signed in
- updated sentence on no-proposal message
- hide faucet while not signed in

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/components/common/TmPage.vue [hunk 1]
```diff
@@ -14,9 +14,9 @@
       <slot slot="header-buttons" name="header-buttons" />
     </tm-page-header>
     <main class="tm-page-main">
-      <template v-if="this.$slots['managed-body']">
-        <card-sign-in-required v-if="signinRequired && !session.signedIn" />
-        <tm-data-connecting v-else-if="!loaded && !connected" />
+      <card-sign-in-required v-if="signInRequired && !session.signedIn" />
+      <template v-if="this.$slots['managed-body'] && session.signedIn">
```
Comment (faboweb) [inline app/src/renderer/components/common/TmPage.vue]: what happens if `!signInRequired && !session.signedIn`?

File: test/unit/specs/components/common/__snapshots__/CardSignInRequired.spec.js.snap [hunk 2]
```diff
@@ -12,7 +12,8 @@ exports[`CardSignInRequired shows a sign in required card 1`] = `
    
   <div>
     
-    This page shows data for your address. To access the page, please 
+    This page shows data for a specific address. To access the page, please
+
```
Comment (faboweb) [inline test/unit/specs/components/common/__snapshots__/CardSignInRequired.spec.js.snap]: sth missing?

File: app/src/renderer/components/wallet/PageWallet.vue [hunk 3]
```diff
@@ -25,7 +25,7 @@
       @show-modal="showModal"
     />
     <tm-btn
-      v-if="enableFaucet"
+      v-if="session.signedIn && enableFaucet"
```
Comment (faboweb) [inline app/src/renderer/components/wallet/PageWallet.vue]: I still have my stomach mur when I see hiding content this way.

File: app/src/renderer/components/common/TmPage.vue (hunk 1 above)
Comment (faboweb) [inline app/src/renderer/components/common/TmPage.vue]:
```suggestion
      <template v-else-if="this.$slots['managed-body']">
```

Comment (jbibla) [reply app/src/renderer/components/common/TmPage.vue]: it will render the template

File: app/src/renderer/components/wallet/PageWallet.vue (hunk 3 above)
Comment (jbibla) [reply app/src/renderer/components/wallet/PageWallet.vue]: i appreciate you saying that every time i hide a button 😆

i agree with you but this feature wasn't implemented properly then. what should happen if someone clicks the faucet button when they are not signed in? nothing even happens when the user _is_ signed in 😕

File: app/src/renderer/scripts/boot.js [hunk 4]
```diff
@@ -33,13 +33,6 @@ export const routeGuard = store => (to, from, next) => {
...
     next(to.path)
-  } else if (
-    !store.state.session.signedIn &&
-    to.matched.some(record => record.meta.requiresAuth)
-  ) {
-    // redirect to session page if auth required
-    store.commit(`setSessionModalView`, `welcome`)
-    store.commit(`toggleSessionModal`, true)
   }
   next()
```
Comment (jbibla) [inline app/src/renderer/scripts/boot.js]: does anyone know if this `next()` is needed?

File: test/unit/specs/components/common/__snapshots__/CardSignInRequired.spec.js.snap [hunk 5]
```diff
@@ -12,7 +12,8 @@ exports[`CardSignInRequired shows a sign in required card 1`] = `
    
   <div>
     
-    This page shows data for your address. To access the page, please 
+    This page shows data for a specific address. To access the page, please
+    
     <a>
       sign in
```
Comment (jbibla) [inline test/unit/specs/components/common/__snapshots__/CardSignInRequired.spec.js.snap]: it's here.

File: test/unit/specs/components/common/__snapshots__/CardSignInRequired.spec.js.snap (hunk 2 above)
Comment (jbibla) [reply test/unit/specs/components/common/__snapshots__/CardSignInRequired.spec.js.snap]: https://github.com/cosmos/voyager/pull/2176/files#r262917640

File: app/src/renderer/components/governance/TabProposals.vue [hunk 6]
```diff
@@ -4,7 +4,7 @@
   <tm-data-empty
     v-else-if="Object.keys(proposals.proposals).length === 0"
     title="No Governance Proposals"
-    subtitle="There are currently no governance proposals on the Cosmos Hub. Submit a new one by clicking the 'Create Proposal' button above."
+    subtitle="There are currently no governance proposals to display. Click the 'Create Proposal' button."
```
Comment (jbibla) [inline app/src/renderer/components/governance/TabProposals.vue]:
```suggestion
    subtitle="There are currently no governance proposals to display. Click the 'Create Proposal' button to submit a proposal."
```

File: app/src/renderer/components/wallet/PageWallet.vue (hunk 3 above)
Comment (faboweb) [reply app/src/renderer/components/wallet/PageWallet.vue]: alright

Comment (sabau) [reply app/src/renderer/components/wallet/PageWallet.vue]: gneh, I would put this condition on line 61, so we also test it

Comment (jbibla) [reply app/src/renderer/components/wallet/PageWallet.vue]: done.

File: test/unit/specs/components/wallet/__snapshots__/PageWallet.spec.js.snap [hunk 7]
```diff
@@ -95,10 +144,7 @@ exports[`PageWallet should show the sending modal 1`] = `
     coin="[object Object]"
   />
    
-  <tm-btn-stub
-    color="green"
-    value="Get Tokens"
-  />
+  <!---->
```
Comment (sabau) [inline test/unit/specs/components/wallet/__snapshots__/PageWallet.spec.js.snap]: I was expecting this to be shown, faucet is set and signedIn is true

**Pull request merged**